### DISCUSSION TITLE
Airsim multi vehicle simulation in Unreal environment

### DISCUSSION THREAD

**Discussion by Flautdn** [microsoft/AirSim#4189] [Support Q&A]
I got a car and a drone as vehicles set up in my unreal environment.

I enabled the api control to the drone.
And disabled the api control to the car and gave controls over the keyboard to maneuver in the unreal environment.

Getting the car state and assigning the moveToPositionAsync command to move the drone to the car position is I'm trying to do.
But the position value of the drone does not change.
Is it allowed to disable the api control to car and control the drone through apis in python?

Thanks

**Comment by zimmy87** [comment]
Currently AirSim does not support using both a car and a drone at the same time, the [SimMode](https://microsoft.github.io/AirSim/settings/#simmode) must be configured to be either "Car" or "Multirotor". They can't be combined right now.
